homeassistant/components/assist_satellite/entity.py

```diff
@@ -71,7 +70,7 @@ class AssistSatelliteEntity(entity.Entity):
 
     _run_has_tts: bool = False
     _is_announcing = False
-    _wake_word_intercept_future: asyncio.Future[str | None] | None = None
+    _wake_word_listener: Callable[[str | None, str], None] | None = None
```

Comment:
Why not use a future ? It allows error and result.

Reply:
I assumed you wanted me to use a listener instead :smile:

Reply:
Kept the future